pandas/core/groupby/ops.py

```diff
@@ -348,6 +349,9 @@ def _ea_wrap_cython_operation(
         elif isinstance(values.dtype, FloatingDtype):
             # FloatingArray
             npvalues = values.to_numpy(values.dtype.numpy_dtype, na_value=np.nan)
+        elif isinstance(values.dtype, StringDtype) and self.how in ["last", "first"]:
```

Comment:
are there other functions (e.g. .sum) as well (or just have *all* string functions hit this path)? do we have sufficient asv's to cover this?

Reply:
yes, other functions do hit this path.
asv coverage is little spotty, i was not able to find anything with groupby and StringDtype

Reply:
> yes, other functions do hit this path.

ok which ones? what if you remove the self.how entirely

Reply:
i don't really like the self.how here, this is way too specific.